common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -206,3 +213,131 @@ def test_all_modes_for_courses(self):
+        ('honor', False),
+        ('audit', False)
+    )
+    @ddt.unpack
+    def test_is_verified_slug(self, mode_slug, is_verified):
+        # check that mode slug is verified or not
+        if is_verified:
+            self.assertTrue(CourseMode.is_verified_slug(mode_slug))
+        else:
+            self.assertFalse(CourseMode.is_verified_slug(mode_slug))
+
+    @ddt.data(
+        ("verified", "verify_need_to_verify"),
+        ("verified", "verify_submitted"),
+        ("verified", "verify_approved"),
+        ("verified", 'dummy'),
+        ("verified", None),
+        ('honor', None),
+        ('honor', 'dummy'),
+        ('audit', None),
+        ('professional', None),
+        ('no-id-professional', None),
+        ('no-id-professional', 'dummy')
+    )
+    @ddt.unpack
+    def test_get_certificate_display(self, mode, verification_status):
+        if mode == "verified":
+            if verification_status in ['verify_need_to_verify', 'verify_submitted']:
+                self.assertEqual(
+                    CourseMode.get_certificate_display(mode, verification_status),
+                    {
+                        "enrollment_title": "Your verification is pending",
+                        "enrollment_value": "Verified: Pending Verification",
+                        "show_image": True,
+                        "image_alt": 'ID verification pending',
+                        "display_mode": 'verified'
+                    }
+                )
+
+            if verification_status in ['verify_approved']:
+                self.assertEqual(
+                    CourseMode.get_certificate_display(mode, verification_status),
+                    {
+                        "enrollment_title": "You're enrolled as a verified student",
+                        "enrollment_value": "Verified",
+                        "show_image": True,
+                        "image_alt": 'ID Verified Ribbon/Badge',
+                        "display_mode": 'verified'
+                    }
+                )
+
+            if verification_status is None:
+                self.assertEqual(
+                    CourseMode.get_certificate_display(mode, verification_status),
+                    {
+                        "enrollment_title": "You're enrolled as an honor code student",
+                        "enrollment_value": "Honor Code",
+                        "show_image": False,
+                        "image_alt": '',
+                        "display_mode": 'honor'
+                    }
+                )
+        if mode == "honor":
+            self.assertEqual(
+                CourseMode.get_certificate_display(mode, verification_status),
+                {
+                    "enrollment_title": "You're enrolled as an honor code student",
+                    "enrollment_value": "Honor Code",
+                    "show_image": False,
+                    "image_alt": '',
+                    "display_mode": 'honor'
```

Comment:
Instead of using raw dictionaries like these, you might want to consider using a helper function to generate these dictionaries for you, in case something about them changes in the future. At the very least, you should store these dictionaries as constants, so that you don't duplicate test data unnecessarily.